src/deploy.sh

```diff
@@ -391,11 +391,11 @@ function cleanup()
 
   if [[ -v options_values['REMOTE'] ]]; then
```

Comment:
Actually, this if is the problem... idk why we added it... anyway, could you update your PR and remove this `if`?

Reply:
When deploy and build were together, when interrupting/signal handling, we would check if the current command was a deploy or a build before cleaning. Somehow this test got in the main cleanup function, probably during some of the recent deploy refactorating